stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -1706,10 +1701,10 @@ void SetUpInterfacesInterfaceEthernetStateAutoNegotiate(
 // data from the DataResponse proto received from SwitchInterface, i.e.,
 // "&DataResponse::PortCounters::message", where message field in
 // DataResponse::Counters.
-TreeNodeEventHandler GetPollCounterFunctor(
-    uint64 node_id, uint32 port_id,
-    ::google::protobuf::uint64 (PortCounters::*func_ptr)() const,
-    YangParseTree* tree) {
+TreeNodeEventHandler GetPollCounterFunctor(uint64 node_id, uint32 port_id,
+                                           uint64 (PortCounters::*func_ptr)()
+                                               const,
```

Comment:
is there a better way to make `const` and the `func_ptr` in the same line?

Reply:
Yes! I think the yang tree arg should come before the func_ptr, like with all the other `Get*Functor` functions.